src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -1843,27 +1818,13 @@ public function testPhpConstantTagMappingKey()
         $this->assertSame($expected, $this->parser->parse($yaml, Yaml::PARSE_CONSTANT));
     }
 
-    public function testPhpConstantTagMappingKeyWithKeysCastToStrings()
```

Comment:
IMO, this test should not disappear. We need to keep it to ensure this case keeps working in 3.4 (it should become a legacy test though)